Sources/Amplitude/AMPUtils.m

```diff
@@ -187,9 +187,11 @@ + (CGFloat)statusBarHeight {
 
 + (UIWindow *)getKeyWindow {
     if (@available(iOS 13.0, *)) {
-        for (UIWindow *window in [[AMPUtils getSharedApplication] windows]) {
```

Comment:
I think windows is [deprecated in iOS 15](https://www.reddit.com/r/SwiftUI/comments/p9gxw6/windows_was_deprecated_in_ios_15_any_help_on/), but here is for iOS 13.0. 
Do we need to do any changes here?

Reply:
I think we are good here. There are two `windows`:
- https://developer.apple.com/documentation/uikit/uiapplication/1623104-windows?language=objc, this one deprecates in iOS 15
- https://developer.apple.com/documentation/uikit/uiwindowscene/3198091-windows?language=objc, this one starts from iOS 13, is the recommended one